### DISCUSSION TITLE
Github blame view shows html markup

### DISCUSSION THREAD

**Discussion by snusmu** [community/community#22190] [Projects and Issues]
<p>github blame view seems to be unreadable, shows html markup</p>

**Comment by nethgato** [comment]
<p>Hey <a class="mention" href="/snusmu">@snusmu</a> :wave:</p>
<p>Similarly to <a href="https://github.community/t/gist-embeds-broken/149821" class="inline-onebox">Gist Embeds Broken?</a>, we are experiencing an issue that does in fact seem related.</p>
<p>Our dev team is actively working on a resolution and as I mentioned in the linked thread above, we hope to have an update shortly.</p>
<p>Thanks so much for raising this!</p>

**Comment by nethgato** [comment, marked as answer]
<p>Hey, all! We should be good now!</p>
<p>Please feel free to reply back if that’s not the case for you. :bow:</p>
